### PR TITLE
Ref/44181

### PR DESCRIPTION
**Pull request by boddy-devbranch**

### CODE REVIEW

File: modules/living_spaces_users/src/Plugin/views/filter/LivingSpacesIsCurrentUserInField.php [hunk 1]
```diff
@@ -74,16 +76,29 @@ public function query() {
...
+        $field = trim($field);
+        if (!empty($field)) {
+          $query->addWhere($orGroup, $field, $current_user_id, '=');
+        }
       }
     }
-    $query->addWhere($this->options['group'], $condition);
+    else {
+      $condition = new ConditionGroup('OR');
+      //    $condition = ($query->createConditionGroup('OR'));
```
Comment (victor-aston) [inline modules/living_spaces_users/src/Plugin/views/filter/LivingSpacesIsCurrentUserInField.php]: do we need that line?

File: modules/living_spaces_users/src/Plugin/views/filter/LivingSpacesIsCurrentUserInField.php [hunk 2]
```diff
@@ -74,16 +76,29 @@ public function query() {
...
-
     $current_user_id = $this->currentUser->id();
-    $condition = ($query->createConditionGroup('OR'));
-    foreach (explode(PHP_EOL, $this->options['field_is_current_user']) as $field) {
-      $field = trim($field);
-      if (!empty($field)) {
-        $condition->addCondition($field, $current_user_id, '=');
+
+    if ($query instanceof ExtendedSql) {
+      $orGroup = $query->setWhereGroup('OR', $this->options['group']);
```
Comment (victor-aston) [inline modules/living_spaces_users/src/Plugin/views/filter/LivingSpacesIsCurrentUserInField.php]: please, use `$or_group`

File: modules/living_spaces_group/src/Plugin/views/field/LivingSpacesGroupCurrentUserMember.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @ViewsField("living_spaces_group_current_user_member")
+ */
+class LivingSpacesGroupCurrentUserMember extends FieldPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render(ResultRow $values) {
+    $user = \Drupal::service('entity_type.manager')->getStorage('user')
```
Comment (victor-aston) [inline modules/living_spaces_group/src/Plugin/views/field/LivingSpacesGroupCurrentUserMember.php]: dependency injection

File: modules/living_spaces_group/src/Plugin/views/filter/LivingSpacesGroupInUserJoinedSpaces.php [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        }
+      }
+
+      $this->query->addWhere(
+        $this->options['group'],
+        $this->tableAlias . '.' . $this->realField,
+        $joined_spaces,
+        $operator
+      );
+
```
Comment (victor-aston) [inline modules/living_spaces_group/src/Plugin/views/filter/LivingSpacesGroupInUserJoinedSpaces.php]: empty line

File: modules/living_spaces_menus/config/rewrite/views.view.spaces.yml [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+          field: group_content_id
+          relationship: none
+          group_type: group
+          admin_label: 'Group content'
+          entity_type: group
+          plugin_id: group_to_group_content
+          required: false
+          group_content_plugins:
+            group_membership: group_membership
+            'dip_chat_appointment:chat_appointment': '0'
```
Comment (victor-aston) [inline modules/living_spaces_menus/config/rewrite/views.view.spaces.yml]: we don't have that plugin in our profile.

File: modules/living_spaces_menus/content/menu_link_content/4d89df5f-b766-4abc-a882-feeb218d61e2.yml [hunk 6]
```diff
@@ -37,12 +38,8 @@ default:
   menu_item_roles:
     -
       target_id: administrator
-    -
-      target_id: assistant_to_the_office_manager
```
Comment (victor-aston) [inline modules/living_spaces_menus/content/menu_link_content/4d89df5f-b766-4abc-a882-feeb218d61e2.yml]: should it be removed?

**Review by victor-aston — APPROVED** [review]

**Review by andrew-sereda — APPROVED** [review]

**Pull request merged**